features/cli/resources.feature

```diff
@@ -14,10 +14,10 @@ Feature: resouces related scenarios
     And the output should match:
       | Name:\\s+database            |
       | Selector:\\s+name=database   |
-    When I run the :export client command with:
-      | resource     | svc           |
-      | name         | database      |
-      | o            | yaml          |
+    When I run the :get client command with:
+      | resource      | svc      |
+      | resource_name | database |
+      | output        | yaml     |
```

Comment:
I wonder if using the `--export` option will be better here.

Reply:
It's not support export command in 4.x now

Reply:
```
🐚 oc get -h | grep export
      --export=false: If true, use 'export' for the resources.  Exported resources are stripped of cluster-specific information.
```

Reply:
Oh, I misunderstood  you, I thought "oc export xxx", I will update it later.

Reply:
@akostadinov  Updated it with export option, pls review it, thanks.

Reply:
Thank you!